src/Arcus.WebApi.Unit/Correlation/TelemetryCorrelationTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Arcus.WebApi.Unit.Hosting;
+using Newtonsoft.Json;
+using Serilog.Events;
+using Xunit;
+using static Arcus.WebApi.Unit.Correlation.CorrelationController;
+
+namespace Arcus.WebApi.Unit.Correlation
+{
+    public class TelemetryCorrelationTests
+    {
+        private readonly TestApiServer _testServer = new TestApiServer();
+
+        [Fact]
+        public async Task SendRequest_WithSerilogCorrelationRequestLogging_ReturnsOkWithEncrichedCorrelationLogProperties()
```

Comment:
Can we extend it with:
- A test to verify that it uses 2 different operation ids but same transaction id?
- A test to verify that it uses 2 different operation ids & transaction id?

Reply:
Yes, thanks, that would make it indeed better.

Reply:
That was added.